dashboard-api/docker-compose-common.yaml

```diff
@@ -0,0 +1,22 @@
+services:
+  cockroachdb:
+    image: cockroachdb/cockroach:v21.2.3
+    entrypoint: cockroach start-single-node --insecure --advertise-addr localhost
+    ports:
+      - "26257:26257"
+      - "8080:8080"
+    networks:
+      - dashboard-api
+  setup_cockroachdb:
+    image: cockroachdb/cockroach:v21.2.3
+    volumes:
+      - ./setup_db.sh:/setup_db.sh
+      - ./seed_cockroachdb.sql:/seed_cockroachdb.sql
```

Comment:
This is another opinion, but I always thought that placing bash scripts at the `root` of the OS was not the best idea in terms of security.

Reply:
In this particular case, I can't come up with an attack vector where having this file at the root can become an issue. Do you had something in mind?

Anyways I ended up inlining the script i the docker-compose file 👌

Reply:
IMO, scripts and data (_malicious_) in `/` can access all files in `/` recursively (including `bin`, `lib`, `sys`, `tmp`), and perform actions on those files, such as deleting files, etc.

Unless you are _sensitive_ and _vigilant_ about folder and file permissions in `/`, changing OS-dependent dirs (using `chmod`), could create OS-level side effects. 

On the other hand, if you place all app files in `/{DIR_NAME}` and set the dir permissions appropriately, you get a logical _sandbox_ for all files in the app. The app then, will be _self-contained_ and living in a single place, without any other file or dir that is not _relevant_ to the it.

Reply:
Yes you're right, thanks for the examples 👍